pkg/security/security_profile/activity_tree/activity_tree.go

```diff
@@ -81,6 +81,18 @@ type ActivityTreeOwner interface {
 	NewProcessNodeCallback(p *ProcessNode)
 }
 
+type cookieSelector struct {
+	execTime int64
+	cookie   uint32
+}
+
+func (cs *cookieSelector) isSet() bool {
+	if cs.execTime != 0 && cs.cookie != 0 {
```

Comment:
you could return `cs.execTime != 0 && cs.cookie != 0`

Reply:
Suggestion applied, thanks